src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="container"
+      action="http://localhost:8080/"
+      method="get"
+      data-qa="big"
+    >
+      <label>
+        <img
+          src="/images/Search.svg"
+          alt="Search"
+          class="image image-big"
+        >
```

Comment:
add non-content images through css, not html, so user don't listen to it through screenreader

Reply:
done